x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authc/kerberos/KerberosRealmSettings.java

```diff
@@ -27,6 +27,8 @@
             Setting.simpleString("keytab.path", Property.NodeScope);
     public static final Setting<Boolean> SETTING_KRB_DEBUG_ENABLE =
             Setting.boolSetting("krb.debug", Boolean.FALSE, Property.NodeScope);
+    public static final Setting<Boolean> SETTING_STRIP_REALM_NAME =
+            Setting.boolSetting("strip.realm_name", Boolean.FALSE, Property.NodeScope);
```

Comment:
what about `include_realm_name` with a default of true? or `remove_realm_name` with a default of false? @tvernum thoughts?